src/App.tsx

```diff
@@ -1,26 +1,50 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useDispatch } from 'react-redux';
+import { todosSlice } from './features/todos';
+import { useAppSelector } from './hooks/useAppSelector';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [isLoading, setIsLoading] = useState(false);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const dispatch = useDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const currnetTodo = useAppSelector(state => state.currentTodo);
```

Comment:
There is a typo in the variable name `currnetTodo`. It should be `currentTodo` to match the state selector and avoid potential runtime errors.